portability-api/src/main/java/org/dataportabilityproject/webapp/SetupHandler.java

```diff
@@ -180,9 +183,7 @@ JsonObject handleCopySetup(Headers requestHeaders, PortabilityJob job) {
 
     }
 
-    return Json.createObjectBuilder().add(JsonKeys.DATA_TYPE, job.dataType())
-        .add(JsonKeys.EXPORT_SERVICE, job.exportService())
-        .add(JsonKeys.IMPORT_SERVICE, job.importService()).build();
+    return new DataTransferResponse(job.exportService(), job.importService(), job.dataType(), Status.INPROCESS, "/startCopy");
```

Comment:
Either now or as a TODO, let's make /startCopy a constant in a shared class so we can keep all our urls in a central place

Reply:
Ack - I'll do that in my next PR